app/models/course_lesson.rb

```diff
@@ -7,4 +7,8 @@ class CourseLesson < ApplicationRecord
 
   validates :title, presence: { message: "Enter a title" }
   validates :content, presence: { message: "Enter content" }
+
+  def content_in_html
```

Comment:
Why not just call this to_html ?

Reply:
Maybe `content_to_html`? 

There is a bunch of elements on lesson that won't be in this html - like title, previous / next lessons, module. Just `to_html` sounds like it would want to include them.